server/util.go

```diff
@@ -238,20 +241,62 @@ func rpcConnect(addr string) (net.Conn, error) {
 	return nil, errors.Errorf("connect to %s failed", addr)
 }
 
+type redirectFormatter struct {
+	buf bytes.Buffer
+}
+
+// Format turns capnslog logs to ngaut logs.
+// TODO: remove ngaut log caller stack, "util.go:xxx"
+func (rf *redirectFormatter) Format(pkg string, level capnslog.LogLevel, depth int, entries ...interface{}) {
+	rf.buf.WriteString(fmt.Sprint(level.Char(), " | "))
+
+	if pkg != "" {
+		rf.buf.WriteString(pkg + ": ")
+	}
+	str := fmt.Sprint(entries...)
+	endsInNL := strings.HasSuffix(str, "\n")
```

Comment:
Sorry I miss one word in last comment.

It checks if there is a `\n` at the end of log, if not, it appends one.